DashKitUI/Protocols/PodDetails .swift

```diff
@@ -0,0 +1,18 @@
+//
+//  PodDetails .swift
+//  DashKitUI
+//
+//  Created by Pete Schwamb on 4/14/20.
+//  Copyright © 2020 Tidepool. All rights reserved.
+//
+
+import Foundation
+
+protocol PodDetails {
+    var podIdentifier: String { get }
+    var lot: String { get }
```

Comment:
[nit] maybe it is kinda obvious to others, but if these members are acronyms or shorthand, it would sure help me if they were spelled out, or at least add comments as to what they represent.  Thanks :)

Reply:
They're not not acronyms.  They're the actual names of the fields that the omnipod PDM displays.  I think these terse fields are going away anyways; will be tackled in the work in https://tidepool.atlassian.net/browse/LOOP-1174.